app/views/spree/admin/premade_carts/_add_to_premade_cart_link.html.erb

```diff
@@ -1,8 +1,8 @@
 <% premade_cart = Spree::PremadeCart.first %>
 <div data-hook='admin_product_create_cart'>
   <% if premade_cart %>
-    <%= link_to t('premade_carts.add_to_premade_cart'), edit_admin_premade_cart_path(premade_cart, 'variant_ids[]' => @product.master.id), class: 'btn btn-default w-100' %>
+    <%= link_to I18n.t('premade_carts.add_to_premade_cart'), edit_admin_premade_cart_path(premade_cart, 'variant_ids[]' => @product.master.id), class: 'btn btn-default w-100' %>
```

Comment:
if the old `t` method was a stand-in for `Spree.t` (I am not sure) then this change needs to add the proper scope to the call, thus making it more like `I18n.t('spree.premade_carts.add_to_premade_cart')`

Reply:
I ran into this on another solidus extension, so that's why it caught my eye.

Reply:
The easiest way to test this would be to visit the admin in development mode and see if the strings have terrible text colors. That's usually a good indicator that their i18n fill-ins could not be found.

Reply:
I re-read the locale file, and this is correct as written.